ci-operator/step-registry/ipi/install/vsphere/registry/ipi-install-vsphere-registry-commands.sh

```diff
@@ -6,5 +6,26 @@ set -o pipefail
 
 export KUBECONFIG=${SHARED_DIR}/kubeconfig
 
+
+function operator_progress() {
+
+while true; do
+	progress=$(oc -o yaml get clusteroperators.config.openshift.io "$1" -o go-template='{{ range .status.conditions -}}{{if eq .type "Progressing" -}}{{.status}}{{end -}}{{end -}}')
+	if [[ "${progress}" == "True" ]]; then
+        echo "$(date -u --rfc-3339=seconds) - Not rolled out, sleeping for 15 seconds"
+		sleep 15
+    else
+		break
+	fi
+done
+}
+
+
+echo "$(date -u --rfc-3339=seconds) - Configuring image registry with emptyDir..."
 oc patch configs.imageregistry.operator.openshift.io cluster --type merge --patch '{"spec":{"managementState":"Managed","storage":{"emptyDir":{}}}}'
 
+
+echo "$(date -u --rfc-3339=seconds) - Wait for rollout..."
+operator_progress image-registry
```

Comment:
this is risky.  you might catch the operator still in progressing=false because it hasn't even picked up your config change yet, not because it's done rolling it out.